test/powershell/Modules/Microsoft.Powershell.Host/Start-Transcript.Tests.ps1

```diff
@@ -129,6 +140,35 @@ Describe "Start-Transcript, Stop-Transcript tests" -tags "CI" {
         # Nothing should have been returned by the FileSystemWatcher
         Receive-Job $job | Should -Be $null
     }
+    It "Should keep the existing file's encoding if the 'Append' parameter is used"{
+        $transcriptFilePath = Join-Path $TestDrive ([System.IO.Path]::GetRandomFileName())
```

Comment:
better to put this in `BeforeAll{}` since it's reused?

Reply:
I've used the $transcriptFilePath that already existed in the BeforeAll instead of creating a new one

Reply:
@Gimly Please look CI fails.